lib/compat/wordpress-6.2/class-gutenberg-rest-pattern-directory-controller-6-2.php

```diff
@@ -0,0 +1,140 @@
+<?php
+// TODO: add docs/comments..
+
+class Gutenberg_REST_Pattern_Directory_Controller_6_2 extends Gutenberg_REST_Pattern_Directory_Controller_6_0 {
+	/**
+	 * Registers the necessary REST API routes.
+	 *
+	 * @since 5.8.0
+	 * @since 6.2.0 Added pattern directory categories endpoint.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/categories',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_pattern_categories' ),
+					'permission_callback' => array( $this, 'get_items_permissions_check' ),
+				)
+			),
+		);
+
+		parent::register_routes();
+	}
+
+	public function get_pattern_categories() {
```

Comment:
This fixes the error that prevents categories from loading

```suggestion
	public function get_pattern_categories( $request ) {
```